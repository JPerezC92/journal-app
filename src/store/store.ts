import { configureStore } from "@reduxjs/toolkit";
import { uiReducer, authReducer, notesReducer } from "../reducers";

export const reducers = {
  authReducer,
  uiReducer,
  notesReducer,
};

export const store = configureStore({
  reducer: reducers,
});

export type RootState = ReturnType<typeof store.getState>;
export type AppDispatch = typeof store.dispatch;
