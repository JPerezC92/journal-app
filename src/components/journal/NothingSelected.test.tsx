import { Provider } from "react-redux";
import { MemoryRouter } from "react-router-dom";
import renderer from "react-test-renderer";
import { store } from "../../store/store";
import NothingSelected from "./NothingSelected";

describe("Test on <NothingSelected />", () => {
  test("should match the snapshot correctly", () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <MemoryRouter>
            <NothingSelected />
          </MemoryRouter>
        </Provider>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
