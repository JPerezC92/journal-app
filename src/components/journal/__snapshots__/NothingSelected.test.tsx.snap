// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test on <NothingSelected /> should match the snapshot correctly 1`] = `
<div
  className="nothing__main-content"
>
  <p>
    Select something
    <br />
    or create an entry
  </p>
  <i
    className="far fa-star fa-4x mt-5"
  />
</div>
`;
